third_party/BUILD

```diff
@@ -41,10 +41,11 @@ filegroup(
 cc_library(
     name = "libuuid",
     srcs = ["empty.cc"],
-    linkopts = select({
-        "@bazel_tools//src/conditions:darwin": [],
-        "//conditions:default": ["-luuid"],
-    }),
+    #linkopts = select({
+        #"@bazel_tools//src/conditions:darwin": [],
+        #"//conditions:default": ["-luuid"],
+    #}),
+    deps = [ "@libossp_uuid.dev//:lib" ],
```

Comment:
Seems likely that this will break building on macOS?

Reply:
Yeah, I don't know what to do about macOS, since I don't have one to try it out, and building kythe on free infra exhausts my quota eventually.

Reply:
Reverted.